tests/infrastructure/instance_types/supported_os/conftest.py

```diff
@@ -1,40 +1,53 @@
 import pytest
-from ocp_resources.data_source import DataSource
 
-from utilities.constants import DATA_SOURCE_NAME
-from utilities.storage import data_volume_template_with_source_ref_dict
-from utilities.virt import VirtualMachineForTests
+from tests.infrastructure.instance_types.supported_os.utils import _golden_image_vm_with_instance_type
 
 
 @pytest.fixture(scope="class")
 def skip_if_rhel8(instance_type_rhel_os_matrix__module__):
-    current_rhel_name = [*instance_type_rhel_os_matrix__module__][0]
-    if current_rhel_name == "rhel-8":
+    if [*instance_type_rhel_os_matrix__module__][0] == "rhel-8":
         pytest.xfail("EFI is not enabled by default before RHEL9")
 
 
+@pytest.fixture(scope="class")
+def skip_if_centos(instance_type_centos_fedora_os_matrix__module__):
```

Comment:
rename function to use xfail just to be in sync with other xfail fixtures in repo
ditto with skip_if_rhel8